lib/pco_api/people/address.ex

```diff
@@ -0,0 +1,3 @@
+defmodule PcoApi.People.Address do
+  use PcoApi.Actions
+end
```

Comment:
![image](https://cloud.githubusercontent.com/assets/8367129/15382644/e08eba90-1d40-11e6-9520-9e42315aca48.png)

We should probably eventually @doc `get/0`, `get/1`, etc